src/main.ts

```diff
@@ -107,10 +107,6 @@ async function bootstrap() {
     .configure(ethConnectUrl, fftmUrl, username, password, passthroughHeaders, blockchainRetryCfg);
   app.get(AbiMapperService).configure(legacyERC20, legacyERC721);
 
-  if (autoInit.toLowerCase() !== 'false') {
```

Comment:
Might as well completely remove the parsing of `AUTO_INIT` up on line 81, and remove it from the example `.env` file, since it's no longer used.